app/client/src/git/ce/store/types.ts

```
@@ -0,0 +1,3 @@
+export interface GitArtifactAPIResponsesReduxState {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)